doc/customization/components/hvplot_holoviews.md

```diff
@@ -0,0 +1,60 @@
+# hvPlot and HoloViews
+
+hvPlot and HoloViews are probably the two most used plotting libraries with Panel. So its important to know who to style these.
```

Comment:
To be honest, I'm not sure that's true. On GoatCounter the Bokeh and Plotly pane reference pages have more visits than the HoloViews one, for example